modules/restore-replace-crashlooping-etcd-member.adoc

```diff
@@ -0,0 +1,125 @@
+----
++
+// TODO: Confirm sh-4.4# prompt is okay.
+
+.. Move the etcd data directory to a different location:
++
+----
+sh-4.4# mv /var/lib/etcd/ /tmp
+----
++
+You can now exit the node shell.
+
+. Remove the unhealthy member.
+
+.. Choose a Pod not on the affected node:
++
+In a terminal that has access to the cluster as a `cluster-admin` user, run the following command:
++
+----
+$ oc get pods -n openshift-etcd | grep etcd
+etcd-ip-10-0-131-183.ec2.internal                3/3     Running     0          123m
+etcd-ip-10-0-164-97.ec2.internal                 3/3     Running     0          123m
+etcd-ip-10-0-154-204.ec2.internal                3/3     Running     0          124m
+----
+
+.. Connect to the running etcd container, passing in the name of a Pod that is not on the affected node:
++
+In a terminal that has access to the cluster as a `cluster-admin` user, run the following command:
++
+----
+$ oc rsh -n openshift-etcd etcd-ip-10-0-154-204.ec2.internal
+----
+
+.. View the member list:
++
+----
+sh-4.2# etcdctl member list -w table
+
++------------------+---------+------------------------------+---------------------------+---------------------------+
+|        ID        | STATUS  |             NAME             |        PEER ADDRS         |       CLIENT ADDRS        |
++------------------+---------+------------------------------+---------------------------+---------------------------+
+| 62bcf33650a7170a | started | ip-10-0-131-183.ec2.internal | https://10.0.131.183:2380 | https://10.0.131.183:2379 |
+| b78e2856655bc2eb | started |  ip-10-0-164-97.ec2.internal |  https://10.0.164.97:2380 |  https://10.0.164.97:2379 |
+| d022e10b498760d5 | started | ip-10-0-154-204.ec2.internal | https://10.0.154.204:2380 | https://10.0.154.204:2379 |
++------------------+---------+------------------------------+---------------------------+---------------------------+
+----
+
+.. Remove the unhealthy etcd member, providing the ID:
++
+----
+sh-4.2# etcdctl member remove 62bcf33650a7170a
+Member 62bcf33650a7170a removed from cluster ead669ce1fbfb346
+----
+
+.. Verify that the member was removed:
++
+----
+sh-4.2# etcdctl endpoint health --cluster
+https://10.0.154.204:2379 is healthy: successfully committed proposal: took = 15.387875ms
+https://10.0.164.97:2379 is healthy: successfully committed proposal: took = 17.773205ms
+----
+
+. Force etcd redeployment.
++
+In a terminal that has access to the cluster as a `cluster-admin` user, run the following command:
++
+----
+$ oc patch etcd cluster -p='{"spec": {"forceRedeploymentReason": "single-master-recovery-'"$( date --rfc-3339=ns )"'"}}' --type=merge <1>
+----
+<1> The `forceRedeploymentReason` value must be unique, which is why a timestamp is appended.
++
+// TODO: Add a statement similar to what we used for the other DR procedure: "When the etcd cluster Operator performs a redeployment, the existing nodes are started with new Pods similar to the initial bootstrap scale up."
```

Comment:
@alaypatel07 The sentence I mention here is what we have after forcing etcd deployment in the new restore procedure here [1]. Can we adjust this sentence to add here? Would this be accurate: 

"When the etcd cluster Operator performs a redeployment, a new member is started with new Pods similar to the initial bootstrap scale up."

or just?

"When the etcd cluster Operator performs a redeployment, a new member is started similar to the initial bootstrap scale up."

[1] https://github.com/openshift/openshift-docs/pull/20053/files#diff-4445e0411093c0b0dc43b767e97b0557R131

Reply:
@bergerhoffer I think the user can stay oblivious to the fact that this happens during scale-up, so I am not able to wrap my head around why we need to mention about the similarity. Maybe something like this:

> "When the etcd cluster Operator performs a redeployment, it will make sure all the master nodes will have a functioning etcd pod"

Reply:
Thanks, added!